grammars/html.cson

```diff
@@ -587,12 +587,69 @@
         'include': '#unquoted-attribute'
       }
     ]
+  'tag-class-attribute':
+    'begin': '\\b(class)\\b\\s*(=)'
+    'captures':
+      '1':
+        'name': 'entity.other.attribute-name.class.html'
+      '2':
+        'name': 'punctuation.separator.key-value.html'
+    'end': '(?<=\'|")|(?=\\s|>)'
```

Comment:
This can be simplified to just `(?=\\s|>)`

Reply:
Done.